### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by tariq87** [hashicorp/terraform-provider-aws#27875]
### Terraform Core Version

0.12.31

### AWS Provider Version

4.18.0

### Affected Resource(s)

*aws_elasticache_global_replication_group

### Expected Behavior

- Should be able to add replica to the secondary cluster
```
Error: error updating ElastiCache Replication Group (authzcache-t-redis-qa-secondary): InvalidParameterValue: Cluster [authzcache-t-redis-qa-secondary] is part of a global cluster [sgaui-authzcache-t-redis-qa-global]. Request rejected.
    status code: 400, request id: 07921852-12b0-4575-8f8c-0488f0043ea1
```

### Actual Behavior

Replicas should be added

### Terraform Configuration Files

```terraform
resource "aws_elasticache_replication_group" "secondary" {
  count  = "${var.region=="us-east-2" ? 1:0}"
 # provider = aws.other_region
  provider = "aws.us-east-2"
  replication_group_id       =  "${var.name}-${var.environment}-secondary"
  replication_group_description   =  "${var.name}-${var.environment}"
  security_group_ids   = ["${var.security_group_id}"]
  subnet_group_name    = "${aws_elasticache_subnet_group.subnetgroup.name}"
  global_replication_group_id = "${var.datastore_west2_prefix}-${var.name}-${var.environment}-${var.datastore_suffix}"     

  cluster_mode {
    replicas_per_node_group = 1
  }

  tags = {
    Name = "${var.name}-${var.environment}-secondary"
    "intuit:billing:appenv" = "${var.environment}"
    "intuit:billing:component" = "swimlane ${var.swimlane_name}"
    "intuit:billing:user-app"=  "${var.billing_app_tag}"
    "intuit:asset_id" = "${var.asset}"
    env = "${var.environment}"
    location = "${var.region}"
    swimlane = "${var.swimlane_name}"
    app = "${var.app}"
  }
}
```

### Steps to Reproduce

terraform apply

### Panic Output

Error: error updating ElastiCache Replication Group (authzcache-t-redis-qa-secondary): InvalidParameterValue: Cluster [authzcache-t-redis-qa-secondary] is part of a global cluster [sgaui-authzcache-t-redis-qa-global]. Request rejected.
    status code: 400, request id: 07921852-12b0-4575-8f8c-0488f0043ea1

**Comment by tariq87** [comment]
can someone please help with this.

**Comment by justinretzolk** [comment]
Hey @tariq87 👋 Thank you for taking the time to raise this! Can you verify the version of the AWS Provider that you're running? I saw a similar thing was previously reported with #19929, but it seems to have been fixed in version 3.50.0 with #20111.

**Comment by justinretzolk** [comment]
Since we haven't heard back in a while, I'm going to close this issue out. If you're still having troubles, or have future issues with the AWS Provider, please feel free to open a new issue (referencing this one for context as needed).
